docs/about/contributing.md

```diff
@@ -41,148 +41,158 @@ We use [semantic-release](https://www.npmjs.com/package/semantic-release), which
+* **[datastore-sequelize][datastore-sequelize-repo]**: MySQL, PostgreSQL, SQLite3, and MS SQL implementations
+* **[datastore-dynamodb][datastore-dynamodb-repo]**: DynamoDB implementation, with [dynamic-dynamodb][dynamic-dynamodb-repo] being used to create the datastore tables for it
+
+### Artifacts
 
-### Scms
+The [Artifact Store][store-repo] (not to be confused with the datastores mentioned above) is used for saving log outputs, shared steps, templates, test coverage, and any artifacts that are generated during a build. The [log service][log-service-repo] is a Go tool for reading logs from the Launcher and uploading them to the store. The [artifact-bookend][artifact-bookend-repo] is used for uploading artifacts to the store.
 
-An scm implementation is used as the interface between the API and an scm. There are several implementations written in node around a common interface.
+### Source Code Management
 
-* **[scm-base][scm-base-repo]**: Common interface [![Version][scm-base-npm-image]][scm-base-npm-url]
-* **[scm-bitbucket][scm-bitbucket-repo]**: Bitbucket implementation [![Version][scm-bitbucket-npm-image]][scm-bitbucket-npm-url]
-* **[scm-github][scm-github-repo]**: Github implementation [![Version][scm-github-npm-image]][scm-github-npm-url]
+An SCM implementation is used as the interface between the API and an SCM. There are several implementations written in node around a common interface.
 
-### [Config Parser][config-parser-repo] [![Version][config-parser-npm-image]][config-parser-npm-url]
+* **[scm-base][scm-base-repo]**: Common interface
+* **[scm-bitbucket][scm-bitbucket-repo]**: Bitbucket implementation
+* **[scm-github][scm-github-repo]**: Github implementation
+
+### Templates
+
+Templates are snippets of predefined code that people can use to replace a job definition in their `screwdriver.yaml`. A template contains a series of predefined steps along with a selected Docker image.
+
+* **[templates][templates-repo]**: A repo for all build templates
+* **[template-main][template-main-repo]**: The CLI for validating and publishing job templates
+* **[template-validator][template-validator-repo]**: A tool used by the API to validate a job template
+
+### [Config Parser][config-parser-repo]
 
 Node module for validating and parsing user's `screwdriver.yaml` configurations.
 
-### [Guide][guide-repo]
+### [Guide][guide-repo] & [Homepage][homepage-repo]
 
-This documentation! Everything you ever hoped to know about the Screwdriver project.
+The [Guide][guide-repo] is documentation! Everything you ever hoped to know about the Screwdriver project.
+The [Homepage][homepage-repo] is the basic one-pager that powers [Screwdriver.cd][homepage].
+
+### [UI][ui-repo]
+
+The Ember-based user interface of Screwdriver.
 
 ### Miscellaneous Tools
 
+* **[circuit-fuses][circuit-fuses-repo]**: Wrapper to provide a node-circuitbreaker with callback interface
 * **[client][client-repo]**: Simple Go-based CLI for accessing the Screwdriver API
```

Comment:
Line 140 for `client-repo`
Yeah we should add a deprecated or something somewhere